tests/containers/integration-test-local

```diff
@@ -8,6 +8,7 @@ RUN dnf install --repo bluechi-rpms \
         --repofrompath bluechi-rpms,file:///tmp/bluechi-rpms/ \
         --nogpgcheck \
         --nodocs \
+        --skip-broken \
```

Comment:
We shouldn't add `--skip-broken` here. What is the error you get when building without this flag?